compiler/sem/sigmatch.nim

```diff
@@ -2509,7 +2513,11 @@ proc setSon(father: PNode, at: int, son: PNode) =
 
 # we are allowed to modify the calling node in the 'prepare*' procs:
 proc prepareOperand(c: PContext; formal: PType; a: PNode): PNode =
-  if formal.kind == tyUntyped and formal.len != 1:
+  when defined(nimCompilerStacktraceHints):
+    {.line.}:
+      frameMsg(c.config, a)
```

Comment:
```suggestion
    frameMsg(c.config, a)
```

Reply:
IIRC the lineinfo gets messed up without `{.line.}:` to unwrap each template application.

Reply:
Hm, but since not inside a template, it would just override the line information of `frameMsg(c.config, a)` with the line information from unrelated template or generic, or crash the compiler with an index defect, no?